examples/migration/create_complete_campaign_google_ads_api_only.py

```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example creates a search campaign with the help of Google Ads API only.
+
+This code example is the last in a series of code examples that shows how to
+create a Search campaign using the AdWords API, and then migrate it to the
+Google Ads API one functionality at a time. See other examples in this directory
+for code examples in various stages of migration.
+
+In this code example, the functionality to create campaign budget and search
+campaign have been migrated to the Google Ads API. The rest of the functionality
```

Comment:
In this example, the AdWords API isn't used at all, so please update the comment.